pynetbox/lib/response.py

```diff
@@ -161,7 +161,7 @@ def _parse_values(self, values):
                         v = self.default_ret(v, api_kwargs=self.api_kwargs)
                 self._add_cache((k, v))
             else:
-                self._add_cache((k, v.copy()))
+                self._add_cache((k, v))
```

Comment:
The rest looks good, but what's the reason for dropping copy here?

Reply:
It appears that the reason for the removal of the copy is that local_context_data can be returned as types None or str which do not have the copy method. Leaving the copy when trying to return any empty local_context_data fields then breaks.